recipes/awsebcli/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "awsebcli" %}
+{% set version = "3.18.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a44fb91b3f6b35ea1166200df1954f8dde2228e0e1b37267b2ca6b8fa66809b5
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - botocore >=1.15
+    - cement ==2.10.12,<3
+    - colorama >=0.4
+    - future >=0.16.0
+    - pathspec >=0.5.9
+    - python-dateutil >=2.1
+    - requests >=2.20.1
+    - setuptools >=20.0
+    - semantic_version >=2.8.5
+    - six >=1.11.0
```

From the metadata it should be
```suggestion
    - six >=1.11.0,>1.12.0
```